ackhandler/sent_packet_handler.go

```diff
@@ -333,7 +333,11 @@ func (h *sentPacketHandler) SendingAllowed() bool {
 			h.bytesInFlight,
 			h.congestion.GetCongestionWindow())
 	}
-	return !(congestionLimited || maxTrackedLimited)
+	// Workaround for #555:
+	// Always allow sending of retransmissions. This should probably be limited
+	// to RTOs, but we currently don't have a nice way of distinguishing them.
```

Comment:
Right now I don't know the correct, full solution here. I think this is an appropriate workaround without many downsides for part of the issue so that we can continue working on finding other causes.

I don't want to spend time adding more complexity that we probably won't need once we kill the workaround.